cvmfs/server/cvmfs_server_ingest.sh

```diff
@@ -62,8 +65,23 @@ cvmfs_server_ingest() {
     die "Please set the base directory where to extract the tarball, use -b \$BASE_DIR or --base_dir \$BASE_DIR or don't provide the base directory to simply delete entities from the repository"
   fi
 
+  if [ x"$upstream_type" = xgw ]; then
+
+    if [ $multiple_delete -eq 1 ]; then
```

Comment:
What is the cause of these limitations?

Reply:
I could delete files in two completely different directories, and I would need to take a lease on the common parent of those directory. Correct?

It seems wiser to avoid this complexity and just drive the user to make smaller deletes.

Reply:
Thanks for the clarifications. Makes sense, I agree that this can be avoided now and implemented later if there is an actual need.